src/Maintenance/Maintenance/MaintenanceConfiguration/MaintenanceConfigurationCreateOrUpdateMethod.cs

```diff
@@ -163,6 +232,55 @@ public override void ExecuteCmdlet()
             HelpMessage = "The Recurrence interval.")]
         public string RecurEvery { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "List of packges include during vm patch operation")]
+        public HashSet<string> LinuxParameterPackageNameMaskToInclude { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "List of packages exclude during vm patch operation")]
+        public HashSet<string> LinuxParameterPackageNameMaskToExclude { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "List of linux patch classifications")]
+        public HashSet<string> LinuxParameterClassificationToInclude { get; set; }
```

Comment:
Should we rename all prefixes to plural?
LinuxParameters
WindowsParameters

Reply:
As per powershell design its not allowed , please refer design review issue , we should not use plural